extensions/graph/pkg/identity/ldap.go

```diff
@@ -5,12 +5,15 @@ import (
 	"errors"
 	"fmt"
 	"net/url"
+	"strings"
 
 	"github.com/go-ldap/ldap/v3"
 	"github.com/gofrs/uuid"
 	ldapdn "github.com/libregraph/idm/pkg/ldapdn"
 	libregraph "github.com/owncloud/libre-graph-api-go"
 
+	"golang.org/x/exp/slices"
+
```

Comment:
I don't think we need the empty lines here. I guess the one on line 14 got also there accidently. (At least elsewhere we only split the `import` block into two parts)